content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -188,6 +224,11 @@ quote`.
        :align: center
        :alt: Sample of an uploaded pdf with the inside quote option chosen in Odoo Sales.
 
+Beside the :guilabel:`File Content` field, you have the possibility to
+:guilabel:`Configure dynamic fields`. When doing so, remember that the starting model is the
+:guilabel:`sale_order_line`, unlike for headers and footers that start from the
+:guilabel:`sale_order`.
```

Comment:
Similar to the previous comment, I can't find sale_order_line or sale_order in the UI.